css/style.css

```diff
@@ -0,0 +1,372 @@
+body, html {
+height: 100%;
+width: 100%;
+margin: 0;
+font-family: Rubik; sans-serif;
+font-size: 25px;
+    }
+  
+.main {
+      background-color: gainsboro;
+    }
+
+  /*  Navigation   */
+
+.desktop-nav {
```

Comment:
intend your code beautifully please
<pre>
class
    style
    style
    style
</pre>

not 

<pre>
class
style
         style
   style
</pre>